alerta-covid/app/views/places/index.html.erb

```diff
@@ -1,37 +1,38 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Places</h1>
+<div class="container-fluid">
+  <h1 class="title">Places</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Area</th>
-      <%if user_signed_in? %>
-        <th colspan="4"></th>
-      <%end%>
-    </tr>
-  </thead>
-
-  <tbody>
+  <div class="container-index-places">
     <% @places.each do |place| %>
-      <tr>
-        <td><%= place.name %></td>
-        <td><%= place.area.name %></td>
+      <div class="card shadow card-place">
+        <%= link_to image_tag("qr_codes/qrcode_#{place.id}.png", class: "qr-code"), place %>
```

Comment:
question, this make that when the user click the image the qr code redirect him be redirectect to the place?, is like and image link? the question is because we are searching something similar option to use images with link_to

Reply:
Exactly bro, this creates an image link to show the place. The image is passed as the body parameter of the link_to method.
https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

Reply:
thanks for the info! :D

Reply:
good info, I didn't know you could do that